Updating AWS to use pypi instead of brew

kris-nova opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.kubernetes.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1256)
<!-- Reviewable:end -->

justinsb commented:
This is a little tricky.  Technically we only care that AWS is set up correctly - we don't really use the AWS CLI.  I guess this is better, but I worry we're making it harder to get going.  Is there a huge difference between the two versions?

kris-nova commented:
@justinsb 

In the demo we are using concrete examples with the python AWS cli tool. @chrislovecnm is right, we should offer an installation suggestion for the user if they are following along. The problem was that we were getting feedback that we were suggesting that our users use an unofficial release of the tool. 

I think I fixed everyone's concerns in my most recent commit. I added a note, with examples of both installation avenues.

Cheers
